src/runtime/audience-action-local-flow.ts

```diff
@@ -787,10 +800,29 @@ export class AudienceActionLocalFlow implements AudienceActionFlow {
   }
 
   close() {
-    if (this.params_.action === TYPE_REWARDED_AD) {
+    if (this.params_.action === InterventionType.TYPE_REWARDED_AD) {
       this.closeRewardedAdWall_();
-    } else if (this.params_.action === TYPE_NEWSLETTER_SIGNUP) {
+    } else if (
+      this.params_.action === InterventionType.TYPE_NEWSLETTER_SIGNUP
+    ) {
       this.closeOptInPrompt_();
     }
   }
+
+  private rewardedResult(
```

Comment:
The function name doesn't seem to convey what exactly it does. Can you rename it to be more specific on what it does?